packages/contracts/ts/index.ts

```diff
@@ -1,9 +1,4 @@
-export {
-  getProxyContract,
-  deployProxyClone,
-  type IDeployCloneArgs,
-  type IGetProxyContractArgs,
-} from "@excubiae/contracts";
+export { getProxyContract, deployProxyClone, type IGetProxyContractArgs } from "@excubiae/contracts";
```

Comment:
we are exporting `deployProxyClone` here. Should we delete it so it can be used only with deployment? It could confuse people 😊